api/tests/unit/application/certification-report/index_test.js

```diff
@@ -17,4 +18,35 @@ describe('Unit | Application | Certifications Report | Route', function() {
     // then
     expect(response.statusCode).to.equal(200);
   });
+
+  describe('POST /api/certification-reports/{id}/abort', function() {
+
+    it('allow access if the session of the logged used is authorized', async function() {
+      // given
+      sinon.stub(certificationSessionAuthorization, 'verify').returns('ok');
+      sinon.stub(certificationReportController, 'abort').returns('ok');
+      const httpTestServer = new HttpTestServer();
+      await httpTestServer.register(moduleUnderTest);
+
+      // when
+      const response = await httpTestServer.request('POST', '/api/certification-reports/1/abort');
+
+      // then
+      expect(response.statusCode).to.equal(200);
+    });
+
+    it('denies access if the session of the logged used is not authorized', async function() {
```

Comment:
**Suggestion**: "Returns HTTP 403 if the logged user is not allowed to access the session".